src/ngraph/runtime/gpu/cudnn_emitter.cpp

```diff
@@ -112,13 +112,49 @@ runtime::gpu::CUDNNEmitter::CUDNNEmitter(GPUPrimitiveEmitter* emitter)
 {
 }
 
+cudnnDataType_t runtime::gpu::CUDNNEmitter::get_cudnn_datatype(std::string dtype)
+{
+    static const std::unordered_map<std::string, cudnnDataType_t> datatype_map{
+        {"float", CUDNN_DATA_FLOAT},
+        {"double", CUDNN_DATA_DOUBLE},
+        {"int8_t", CUDNN_DATA_INT8},
+        {"int32_t", CUDNN_DATA_INT32}};
+    auto p = datatype_map.find(dtype);
+    if (p == datatype_map.end())
+    {
+        std::string err = dtype + "is not supported by CuDNN";
```

Comment:
Maybe it doesn't matter, not sure though: lower case C, cuDNN. Same for other references in this file.

Reply:
fixed in all comments.